src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -88,6 +88,11 @@ public void save(AddressBook addressBook) throws StorageOperationException {
         /* Note: Note the 'try with resource' statement below.
          * More info: https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html
          */
+    	if(!path.toFile().exists()){
+    		//throw new StorageOperationException("File does not exist!");
+    		System.out.println("File does not exist! It was created for you.");
```

Comment:
Why is this inappropriate? Which class is responsible for interacting with the user (Hint: contains the word ui)?
Try figuring out which class would be the best to handle the error thrown here 
(Hint: contains an object named `ui` ;-) ).